cpp/dolfinx/fem/Form.h

```diff
@@ -281,6 +293,19 @@ class Form
     return _integrals[static_cast<std::size_t>(type)].size();
   }
 
+  /// @brief Indicator of which coefficient is enabled for a given integral
+  /// (kernel).
+  /// @param[in] type Integral type.
+  /// @param[in] i The index of the integral.
+  std::span<const std::int8_t> enabled_coefficients(IntegralType type,
+                                                    std::size_t i) const
+  {
+    assert(i < _integrals[static_cast<std::size_t>(type)].size());
+    const std::vector<std::int8_t>& enabled_coeffs
+        = _integrals[static_cast<std::size_t>(type)][i].enabled_coefficients;
+    return std::span<const std::int8_t>(enabled_coeffs.data(),
```

Comment:
This looks shaky from a memory safety perspective. 

How big is `enabled_coeffs` likely to be?

Reply:
It is the number of unique coefficients in your form. I would be suprised if this hits more than 100.

Reply:
Could you elaborate on why this is shaky?

Reply:
Spans of objects of objects. 

Since it's small, it would be much simpler to make it a `std::vector`. Negligible cost and memory safe.